src/components/Editor/Preview/index.js

```diff
@@ -126,21 +127,20 @@ class Preview extends PureComponent {
 const {
   addExpression,
   loadObjectProperties,
-  setPreview,
+  updatePreview,
   clearPreview
 } = actions;
 
 export default connect(
   state => ({
     preview: getPreview(state),
     selectedSource: getSelectedSource(state),
-    linesInScope: getInScopeLines(state),
```

Comment:
this prop is now in the action.
We can move `selectedFrameVisible` as well, which simplifies the data dependencies for the component quite a bit

Reply:
It looks like you can remove `getInScopeLines` from the import (line 14) as well then